submissions/dragalur/HTML-CSS-Popup/index.html

```diff
@@ -1,89 +1,187 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
-    <link rel="stylesheet" href="style.css">
-</head>
-<body>
-    <div class="wrapper">
+	<head>
+		<meta charset="UTF-8" />
```

Comment:
All single tags should be without slash "/". Could you please fix it again?  Earlier they were without slashes. I do not understand the reason why do you add these slashes. Slashes are necessary only for HTML4, XHTML specification (lower than HTML5).   https://codebra.ru/en/lessons-html/dating/2/1

Reply:
> All single tags should be without slash "/". Could you please fix it again? Earlier they were without slashes. I do not understand the reason why do you add these slashes. Slashes are necessary only for HTML4, XHTML specification (lower than HTML5). https://codebra.ru/en/lessons-html/dating/2/1

Fixed this, i use vs code and when  turn on autoformate he add slash